controls/radchartview/axes/categoricalaxis.md

```diff
@@ -10,21 +10,44 @@ position: 3
 
 # Categorical Axis
 
+The categorical axis displays a range of categories and the values that determine each category do not define an order. Here is an example of a categorical axis with a few data points.
 
+#### __[XAML] Example 1: Chart with categorical axis__
+{{region xaml-radchart-axes-categoricalaxis_0}}
+	<telerik:RadCartesianChart>
+		<telerik:RadCartesianChart.HorizontalAxis>
+			<telerik:CategoricalAxis/>
+		</telerik:RadCartesianChart.HorizontalAxis>
 
-## 
+		<telerik:RadCartesianChart.VerticalAxis>
+			<telerik:LinearAxis/>
+		</telerik:RadCartesianChart.VerticalAxis>
 
-The categorical axis displays a range of categories and the values that determine each category do not define an order. If a category binding is not specified for the categorical axis, each data point will be in its own category with the category being determined by the position of the data point in the data source. Here is an example of a categorical axis with a few data points. Notice how each data point is its own category on the X axis:
-        
+		<telerik:RadCartesianChart.Series>
+			<telerik:BarSeries>
+				<telerik:BarSeries.DataPoints>
+					<telerik:CategoricalDataPoint Category="Soda" Value="20"/>
+					<telerik:CategoricalDataPoint Category="Beer" Value="40"/>
+					<telerik:CategoricalDataPoint Category="Milk" Value="35"/>
+					<telerik:CategoricalDataPoint Category="Water" Value="40"/>
+				</telerik:BarSeries.DataPoints>
+			</telerik:BarSeries>
+		</telerik:RadCartesianChart.Series>
+	</telerik:RadCartesianChart>
+{{endregion}}
 
-#### __XAML__
+#### __Figure 1: Result from Example 1__
+![RadChartView with CategoricalAxis](images/radchartview-categoricalaxis.png)
 
-{{region radchart-axes-categoricalaxis_0}}
+If a category binding is not specified for the categorical axis, each data point will be in its own category with the category being determined by the position of the data point in the data source. 

```

Comment:
What is category binding? Also, the category setting is specified for the data point, note for the axis. Instead you can try with something like "If a data point category is not defined the series will set a category automatically. The category will be a numeric value determined by the position of the data point in the data source. For example, "1", "2", "3", etc."